BICCN-policies/DRAFT Citation and identifier policy.md

```diff
@@ -0,0 +1,49 @@
+# Policy regarding support of data and resource citation and use of persistent identifiers
+
+## For data submitters/authors
+
+### Persistent identifiers + metadata for BICCN datasets
+- BICCN consortium members will submit their data to an approved BRAIN repository for publication.  
+- Datasets must include a standard set of metadata specified by the BICCN (TODO:  Get list from Carol)
+- Repositories will issue persistent identifiers (PIDs) required by many journals for referencing a dataset in a publication. 
+- If submitted data are to be included in a manuscript, make sure that you consult with the repository to understand how long it takes for data ingest, validation and PID assignment. (TODO:  Reference charter location)
+
+### Data and resource citation policy for BICCN
+
+- **Citing BICCN/BRAIN tools, including the data archives**: We will adopt the use of Research Resource Identifiers (RRID), supported by many journals and widely used across biomedicine.  [RRIDs](https://scicrunch.org/resources) are persistent identifiers for different types of tools-digital resources, antibodies, organisms, cell lines, biosamples and plasmids, that can be used to identify and track the resources used in a scientific paper.  RRIDs appear in the materials and methods section in the following form:  "Data was deposited into the NEMO data archive (RRID:SCR_016152)".  A list of RRIDs for BRAIN supported archives and tools is maintained by the [Neuroscience Information Framework](https://neuinfo.org//).  Note that many of these resources also request that authors include a citation to a publication describing the resource.
+
+- **Citing data deposited in an archive**:  Authors should cite data sets in a manner similar to citing other papers, as per the the [Joint Declaration of Data Citation Principles](https://www.force11.org/datacitationprinciples).  A data citation contains the following information:  authors, date published, title, version, repository, persistent identifier.  
+
+- The data set is cited in the manuscript using the required style, e.g., "Data (Martone et al., 2019) was deposited into the NEMO data archive (RRID:SCR_016152)" 
+
+- The data citation can be listed in the data availability statement:  Martone et al., 2019
+
+- The full citation should be included in the references:  Martone, ME (2019) My data set, V2.0, Neuroscience Multiomics Data Archive, doi:hzi.12345.
```

Comment:
If an archive has a doi ready for a manuscript this makes sense. If there is an accession id but not a DOI, what does the citation look like?